Makefile

```
@@ -1,22 +1,67 @@
+#! /usr/bin/make -f
+
+# Project variables.
+PROJECT_NAME = starport
 DATE := $(shell date '+%Y-%m-%dT%H:%M:%S')
 VERSION = development
 HEAD = $(shell git rev-parse HEAD)
 LD_FLAGS = -X github.com/tendermint/starport/starport/internal/version.Version='$(VERSION)' \
 	-X github.com/tendermint/starport/starport/internal/version.Head='$(HEAD)' \
 	-X github.com/tendermint/starport/starport/internal/version.Date='$(DATE)'
 BUILD_FLAGS = -mod=readonly -ldflags='$(LD_FLAGS)'
+BUILD_FOLDER = ./dist
+BUILD_CMD = ./starport/cmd/starport
 
+## install: Install de binary.
 install:
 	@echo Installing Starport...
 	@go install $(BUILD_FLAGS) ./...
 	@starport version
 
+## build: Build the binary.
+build: clean
```

```suggestion
build:
```

I think we should keep the cached files created by the compiler to make the `build` command faster for the next executions. I don't see why we should clean it before each build.